core/web/apiv2/audit.py

```diff
@@ -0,0 +1,13 @@
+import datetime
+
+from fastapi import APIRouter, HTTPException
+from pydantic import BaseModel, ConfigDict
+
+from core.schemas.audit import TimelineLog
+
+router = APIRouter()
+
+
+@router.get("/timeline/{id:path}")
```

Comment:
In order to not confuse users, we need to detail in the documentation that the whole path will also include the collection name like `entities/{id}`

Reply:
I also wonder how the OpenAPI spec will behave when building language specific client.